include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -9086,6 +9086,23 @@
         {
           "enumerant" : "NoFusionINTEL",
           "value" : "0x800000",
+          "capabilities" : [ "FPGALoopControlsINTEL" ],
+          "extensions" : [ "SPV_INTEL_fpga_loop_controls" ],
+          "version" : "None"
+        },
+        {
+          "enumerant" : "LoopCountINTEL",
+          "value" : "0x1000000",
+          "parameters" : [
+            { "kind" : "LiteralInteger" }
+          ],
+          "capabilities" : [ "FPGALoopControlsINTEL" ],
+          "extensions" : [ "SPV_INTEL_fpga_loop_controls" ],
```

Comment:
Note: This follows the same pattern used by the other loop controls added for this extension, but I think our current best practice is to gate the capability by the extension, then anything else by the capability only.  Is it sufficient to only include the capability here?

Reply:
Indeed, remove extra addition of the capability for all SPV_INTEL_fpga_loop_controls extension entries

Reply:
Thanks, your latest change removed the redundancy, but I think in the opposite direction it should have: I think it should have removed the extension and left the capability.

Reply:
Oh I see, thanks!

Reply:
Thanks, latest version looks great 👍